mysite/search/migrations/0014_populate_icons_for_existing_projects_LATER.py

```diff
@@ -29,11 +29,6 @@ def forwards(self, orm):
         # Populate icons for all existing projects.
         for orm_project in orm['search.project'].objects.all():
             model_project = Project.objects.get(pk=orm_project.pk)
-            print "Populating icon for", model_project
-            model_project.populate_icon_from_ohloh()
-            # We know this is unsafe.
-            # If this fails, it might be because
-            # populate_icon_from_ohloh doesn't exist anymore.
```

Comment:
Generally, you shouldn't modify anything in a `migrations` directory, as all the contents there are auto-generated by South for Django apps that are older than Django 1.7. Instead, use South to do a migration on the command line, which will add a migration file (to be git committed): http://south.readthedocs.org/en/latest/tutorial/part1.html